src/util.ts

```diff
@@ -0,0 +1,6 @@
+import { BASE_URL_DEFAULT } from './constants'
+import { CID } from './types'
+
+export const getGatewayURL = (cid: CID, baseURL = BASE_URL_DEFAULT): string => {
```

Comment:
This actually is not the gateway url^^
For the content at `QmWATWQ7fVPP2EFGu71UkfnqhYXDYH566qy47CnJDgvs8u`,
the gateway url = `https://ipfs.runfission.com/ipfs/QmWATWQ7fVPP2EFGu71UkfnqhYXDYH566qy47CnJDgvs8u`
the server GET route = `https://runfission.com/ipfs/QmWATWQ7fVPP2EFGu71UkfnqhYXDYH566qy47CnJDgvs8u`
(note the `ipfs` subdomain)
They should return the same content (since they're the same CID), but the requests are actually handled by two different servers

Reply:
although it wouldn't be the worst idea to have getter functions for both.

We could have `getGatewayURL` and `getContentURL`.
On the actual client, I'd say default to the `getContentURL` since we have more control of that seeing as it's handled by our api and not the ipfs gateway.